### DISCUSSION TITLE
Download links for models don't work

### DISCUSSION THREAD

**Discussion by elfin888** [geekyutao/Inpaint-Anything#165] [Q&A]
Hi! Thank you so much for creating this fantastic project!! It look absolutely amazing.

I tried to download the pre-trained models but all the Yandex links are dead. Do you happen to have the files hosted anywhere else?

All of these links are dead:
All the data (models, test images, etc.) https://disk.yandex.ru/d/AmdeG-bIjmvSug
Test images from the paper https://disk.yandex.ru/d/xKQJZeVRk5vLlQ
The pre-trained models https://disk.yandex.ru/d/EgqaSnLohjuzAg
The models for perceptual loss https://disk.yandex.ru/d/ncVmQlmT_kTemQ

The best model (Places2, Places Challenge):
curl -L $(yadisk-direct https://disk.yandex.ru/d/ouP6l8VJ0HpMZg) -o big-lama.zip
unzip big-lama.zip

All models (Places & CelebA-HQ):
curl -L $(yadisk-direct https://disk.yandex.ru/d/EgqaSnLohjuzAg) -o lama-models.zip
unzip lama-models.zip

2. Prepare images and masks
Download test images:
curl -L $(yadisk-direct https://disk.yandex.ru/d/xKQJZeVRk5vLlQ) -o LaMa_test_images.zip
unzip LaMa_test_images.zip

Thanks!
